lepidopter-fh/configure.sh

```diff
@@ -3,3 +3,10 @@ set -ex
 
 # Set locatime to UTC
 cp /usr/share/zoneinfo/UTC /etc/localtime
+
+# Checkout lepidopter-update repo
+git clone https://github.com/OpenObservatory/lepidopter-update \
```

Comment:
I think it's best if you include only the `updater.py` script and not the full repository. The repo contains a bunch of stuff we don't really need.